tests/integration/requirements.txt

```diff
@@ -0,0 +1,8 @@
+juju
```

Comment:
Just to confirm, are all of these necessary? I know during a POC sometimes items are added to requirements and then removed subsequently so just double checking 😃

Reply:
juju has been deleted, the others are necessary